serverless.example.yml

```diff
@@ -8,6 +8,12 @@ provider:
   # uncomment this if using a bucket that already exists for deployment files
   # deploymentBucket:
   #   name: my-deployment-bucket
+  logs:
+    restApi:
+      executionLogging: false
+      fullExecutionData: false
+      accessLogging: false
+      format: '{"requestId":"$context.requestId","ip":"$context.identity.sourceIp","caller":"$context.identity.caller","useragent" : "$context.identity.userAgent","requestTime":"$context.requestTime","httpMethod":"$context.httpMethod","resourcePath":"$context.resourcePath","status":"$context.status","protocol":"$context.protocol","responseLength":"$context.responseLength"}'
```

Comment:
Looks good – in lieu of testing this, do you have a screenshot or anything to show what this produces?

Reply:
added to the PR description

Reply:
The only one i'm not certain about is `caller`, which is in all of the examples of this setting, but always has value `-` in my testing.